.gitignore

```diff
@@ -4,3 +4,5 @@
 Doxyfile*
 doxygen_sqlite3.db
 html
+*.out
+*.o
```

Comment:
UnixHostDuino generates output files/executables which we shouldn't put into the repo.